src/pl.x86.json

```diff
@@ -429,9 +429,10 @@
 		{
 			"folder-name": "GedcomLexer",
 			"display-name": "GEDCOM Lexer",
-			"version": "0.4.0.140",
-			"id": "16c38bd8b57ac63508be9ee78f9d94d865a6df0dc17932d91381df840bd1ad94",
-			"repository": "https://sourceforge.net/projects/gedcomlexer/files/GedcomLexer-0.4.0-r140/GedcomLexer-0.4.0-r140-x86.zip",
+			"version": "0.5.0.170",
+			"npp-compatible-versions": "[7.7,]",
```

Comment:
@stanm The type change of Sci_PositionCR just affects the x64 version of the plugin. The 32bit one will be still compatible with older N++ versions so probably you don't need to set this for the 32bit version as the plugin list is just bundled with a new N++ version.